src/installer/corehost.proj

```diff
@@ -45,8 +45,8 @@
       Use IgnoreStandardErrorWarningFormat because Arcade sets WarnAsError and there's an existing
```

Comment:
cc @ViktorHofer
This is the only corehost related file I had to keep under src/installer directory, as it is sharing some targets (related to Version/GitSHA1 etc.) with rest of the installer infra. Looks like something that can be shared via src/tasks/installer in the future?

Reply:
hmm, src/tasks/* only defines tasks. The repo root's Directory.Build.targets would be used to share targets.

Reply:
cc @jkoritzinsky

Reply:
We also have eng/versioning.targets for very similar stuff. (and yup, targets can be converted to tasks. 😉)

Reply:
Right, but as a principle we should only use tasks where pure msbuild isn't sufficient.